inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -518,35 +518,8 @@
             },
             "type": "array"
         },
-        "subject_terms": {
-            "uniqueItems": true,
-            "items": {
-                "type": "object",
-                "properties": {
-                    "source": {
-                        "enum": [
-                            "automatically added based on DCC, PPF, DK",
-                            "submitter",
-                            "INSPIRE"
-                        ],
-                        "type": "string"
-                    },
-                    "term": {
-                        "type": "string"
-                    },
-                    "scheme": {
-                        "enum": [
-                            "INSPIRE",
-                            "arXiv",
-                            "APS",
-                            "PoS",
-                            "Elsevier"
-                        ],
-                        "type": "string"
-                    }
-                }
-            },
-            "type": "array"
+        "field_categories": {
+            "$ref": "elements/fields.json"
```

Comment:
IMHO, in general, for certain type of categories such as the above `fields`, to allow for singular form is only going to cause troubles in the future when we realize that nevertheless the field is always repeatable.